tests/TestCase/ResultSetTest.php

```diff
@@ -146,7 +148,7 @@ public function testDecoratedMethods()
                 $param = $requireParam[$method];
             }
 
-            $return = $returnValues[$method] ?? 'something';
+            $return = $returnValues[$method];
```

Comment:
Apparently this is trying to test decorated methods, but we don't actually decorate all of the methods. Not really sure this is the correct way to test.

Reply:
Please do.

Reply:
Merged your fix.